docs/reference/teal/specification.md

```diff
@@ -44,7 +44,7 @@ Differences between modes include:
 
 Constants are loaded into the environment into storage separate from the stack. They can then be pushed onto the stack by referring to the type and index. This makes for efficient re-use of byte constants used for account addresses, etc.
 
-The assembler will hide most of this, allowing simple use of `int 1234` and `byte 0xcafed00d`. These constants will automatically get assembled into int and byte pages of constants, de-duplicated, and operations to load them from constant storage space inserted.
+The assembler will hide most of this, allowing simple use of `int 1234` and `byte 0xcafed00d`. These constants will automatically get assembled into int and byte pages of constants, de-duplicated, and operations to load them from constant storage space inserted. See [Constants and Pseudo-Ops](constants-and-pseudo-ops) below for this simplified syntax.
```

Comment:
Missing the # in the link. Should be [Constants and Pseudo-Ops](#constants-and-pseudo-ops)

Reply:
Fixed